packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -104,6 +104,37 @@ function apexLanguageServerDatabase(): string | undefined {
   return apexLangServerDbPath;
 }
 
+function lwcTestResults(expectedCwd: string): string {
+  // todo: should this use getRootWorkspacePath instead?
```

Comment:
as discussed..yup for sure.

Reply:
I'm getting a little hung up on trying to remove this expectedCwd param.  It looks to me like the current root ws path is the only thing that is passed in, but some tests fail when I remove it, and the tests are fairly complex so I'm trying to figure out what else needs to be updated to make the tests work.  I'll keep looking at this

Reply:
cool. lemme know if you want to pair on funky tests.